features/accounts/edit/add-profile-photo/add-profile-photo.tsx

```diff
@@ -53,7 +55,6 @@ export const AddProfilePhoto = () => {
 
   const handleAvatarDelete = () => {
     deleteAvatar();
```

Comment:
There's a lack of functionality here. The modal window to confirm deletion should remain open but disabled until the ajax request to delete the avatar completes successfully.
If response is unsuccessful, the modal window should remain open, but enabled.

Reply:
The same is true for AvatarUploader.